yt_dlp/extractor/youtube.py

```diff
@@ -204,12 +204,12 @@
         'INNERTUBE_CONTEXT': {
             'client': {
                 'clientName': 'IOS',
-                'clientVersion': '19.45.4',
+                'clientVersion': '20.02.3',
                 'deviceMake': 'Apple',
                 'deviceModel': 'iPhone16,2',
-                'userAgent': 'com.google.ios.youtube/19.45.4 (iPhone16,2; U; CPU iOS 18_1_0 like Mac OS X;)',
+                'userAgent': 'com.google.ios.youtube/20.02.3 (iPhone16,2; U; CPU iOS 18_3 like Mac OS X;)',
                 'osName': 'iPhone',
-                'osVersion': '18.1.0.22B83',
+                'osVersion': '18.3.0.22D5040d',
```

Comment:
Per https://appledb.dev/firmware/iOS/22D5040d.html, `22D5040d` is iOS 18.3 beta 2. I imagine that YouTube will soon have no hesitation blocking that iOS version, since practically no one uses betas once the final version releases. In my opinion it would be much better to use the latest iOS 18.2.1

```suggestion
                'osVersion': '18.2.1.22C161',
```

I'm not sure how that would affect the user-agent (it would either include `18_2` or `18_2_1`), so perhaps it might be better to just avoid bumping the osVersion for now?

Reply:
it would be `18_2_1` in the UA

Reply:
while bumping to the latest version of everything is a good idea, this is likely only solving the botgate because the dlp userbase is not using it en masse yet